sale_order_product_availability_inline/tests/test_sale_order_product_availability_inline.py

```diff
@@ -10,8 +10,9 @@ def setUpClass(cls):
         super().setUpClass()
         cls.obj_product = cls.env["product.product"]
         cls.obj_partner = cls.env["res.partner"]
-        cls.obj_inventory = cls.env["stock.inventory"]
+        cls.obj_quant = cls.env["stock.quant"]
         cls.partner = cls.obj_partner.create({"name": "Partner test - sopai"})
+
```

Comment:
Don't introduce empty lines in the middle of a method if they don't previously exist